beluga_ros/include/beluga_ros/ndt_ellipsoid.hpp

```diff
@@ -0,0 +1,132 @@
+// Copyright 2024 Ekumen, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef BELUGA_ROS_NDT_ELLIPSOID_HPP
+#define BELUGA_ROS_NDT_ELLIPSOID_HPP
+
+#include <beluga_ros/messages.hpp>
+
+/**
+ * \file
+ * \brief Utilities for NDT I/O over ROS interfaces.
+ */
+
+namespace beluga_ros {
+
+static void use_mean_covariance(const auto& mean, const auto& eigenSolver, auto& marker){
```

Comment:
```suggestion
inline void use_mean_covariance(const auto& mean, const auto& eigenSolver, auto& marker){
```

Reply:
it may be worth to move the definitions to a .cpp file though

Reply:
@Fernando-Sanz you will need to modify the cmake in that case https://github.com/Ekumen-OS/beluga/blob/08e682fe0795952ad01fc991979fe3056a27c29a/beluga_ros/cmake/ament.cmake#L26-L27